ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/ShopperInsightsClient.kt

```diff
@@ -43,6 +44,17 @@ class ShopperInsightsClient internal constructor(
     constructor(context: Context, authorization: String) : this(
         BraintreeClient(context, authorization)
     )
+    /**
+     * @param context: an Android context
+     * @param authorization: a Tokenization Key or Client Token used to authenticate
+     * @param shopperSessionId: the shopper session ID returned from your server SDK request
+     */
+    constructor(context: Context, authorization: String, shopperSessionId: String) : this(
```

Comment:
Could we combine this constructor with the above one by using a default value for `shopperSessionId`?

Reply:
Done 👍